README.md

```diff
@@ -78,26 +78,39 @@ This module exports a single function.
 
 ```js
 const plugin = createPlugin(
-  globals: Object,
+  globals: { [id: string]: string } | (id: string) => string,
   {
     include?: Array,
-    exclude?: Array
+    exclude?: Array,
+    dynamicWrapper?: String
```

Comment:
It might be more powerful to make `dynamicWrapper` a function i.e.
```js
dynamicWrapper?: (variableName: String) => code: String
```
Which defaults to:
```js
name => `Promise.resolve(${name})`
```

Reply:
Now supports both